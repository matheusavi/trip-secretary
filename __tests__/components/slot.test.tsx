import { render } from "@testing-library/react";
import Slot from "../../app/components/slot";

it("renders header unchanged", () => {
  const { container } = render(<Slot location={1} />);
  expect(container).toMatchSnapshot();
});
