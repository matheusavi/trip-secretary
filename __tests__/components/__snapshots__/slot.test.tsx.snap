// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders slot unchanged 1`] = `
<div>
  <div
    aria-label="Slot 1"
    class="z-10 h-10"
    data-drop-target-for-element="true"
    data-testid="slot-1"
    style="grid-row: 1; grid-column: 2;"
  />
</div>
`;
